packages/datadog-plugin-mysql2/test/integration-test/server.mjs

```diff
@@ -0,0 +1,17 @@
+import 'dd-trace/init.js'
+import mysql from 'mysql2'
+
+const conn = {
+  host: 'localhost',
+  user: 'root',
+  database: 'db',
+  port: 3306
+}
+
+const connection = mysql.createConnection(conn)
+
+connection.connect()
+
+connection.query('SELECT NOW() AS now')
```

Comment:
Same here with an `await`?

Reply:
I'd also assume not:

https://www.npmjs.com/package/mysql2#:~:text=%7D%0A)%3B-,Using%20Prepared%20Statements,-With%20MySQL2%20you